champions/src/main/java/me/mykindos/betterpvp/champions/tips/tips/weapons/BoosterWeaponTip.java

```diff
@@ -0,0 +1,31 @@
+package me.mykindos.betterpvp.champions.tips.tips.weapons;
+
+import com.google.inject.Inject;
+import com.google.inject.Singleton;
+import me.mykindos.betterpvp.champions.Champions;
+import me.mykindos.betterpvp.champions.tips.ChampionsTip;
+import me.mykindos.betterpvp.core.components.champions.Role;
+import me.mykindos.betterpvp.core.utilities.UtilMessage;
+import org.bukkit.entity.Player;
+
+@Singleton
+public class BoosterWeaponTip extends ChampionsTip {
+
+    @Inject
+    public BoosterWeaponTip(Champions champions) {
+        super(champions, 1, 1,
+                UtilMessage.deserialize("<gold>Booster</gold> weapons (<gold>Gold</gold> Sword / <gold>Gold</gold> Axe) increase the " +
+                        "sword/axe skill by <yellow>1</yellow> level")
```

Comment:
Don't forget crossbows

You can probably be less specific
Booster weapons increase the level of their corresponding skill by 1 level

Reply:
Done